src/main/java/sc/fiji/bdvpg/services/ISourceAndConverterService.java

```diff
@@ -98,4 +99,6 @@
      */
     List<SourceAndConverter> getSourceAndConvertersFromSource( Source source );
 
+    BdvSourceServiceUI getUI();
```

Comment:
Let's try to keep the UI  separated and just keep actions

Reply:
https://github.com/bigdataviewer/bigdataviewer-playground/pull/75/commits/99f88c9861779dab544fc05b71ac09135e864301 revert this